src/sinol_make/helpers/printer.py

```diff
@@ -0,0 +1,179 @@
+import curses
+
+from datetime import datetime, timedelta
+from curses import wrapper
+
+
+def printer(func, *args, **kwargs):
+    """
+    Prints output of func (called every 0.1 seconds with terminal width and height, then args and kwargs)
```

Comment:
Is there a reason for choosing specifically 0.1? Maybe we can refresh it faster? Probably that would smoothen out the scroll, though it's not necessary.

Reply:
The `func`'s output will change every 0.1 seconds, but the keyboard input is polled as often as possible, so scrolling should be fast. We could change the 0.1 second interval. We can reduce the time interval, but I don't think it will change much visually, but it could slow down scrolling.